src/Exception/InvocationFailureExceptionInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Dhii\Invocation\Exception;
+
+/**
+ * An exception that occurs when an invoker fails to invoke a command.
+ *
+ * @since [*next-version*]
+ */
+interface InvocationFailureExceptionInterface extends CommandInvokerExceptionInterface
```

Comment:
Shouldn't this exception also be aware of the command code, and perhaps even the arguments?